astroquery/dace/core.py

```diff
@@ -0,0 +1,80 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+from collections import defaultdict
+from astropy.table import Table
+from ..query import BaseQuery
+from ..utils import async_to_sync
+from . import conf
+
+__all__ = ['Dace', 'DaceClass']
+
+
+class ParseException(Exception):
+    """Raised when parsing Dace data fails"""
+
+
+"""
+DACE class to access DACE (Data Analysis Center for Exoplanets) data based in Geneva Observatory  
+"""
+
+
+@async_to_sync
+class DaceClass(BaseQuery):
+    __DACE_URL = conf.server
+    __DACE_TIMEOUT = conf.timeout
+    __OBSERVATION_ENDPOINT = 'ObsAPI/observation/'
+    __RADIAL_VELOCITIES_ENDPOINT = __OBSERVATION_ENDPOINT + 'radialVelocities/'
+    __HEADERS = {'Accept': 'application/json'}
+
+    def query_radial_velocities(self, object_name):
+        """
+        Get radial velocities data for an object_name. For example : HD40307
+
+        Parameters
+        ----------
+        object_name : the target you want radial velocities data
+
+        Return
+        ------
+        table : ~astropy.table.Table Table containing radial velocities data for the object_name given
+        """
+        response = self._request("GET", ''.join([self.__DACE_URL, self.__RADIAL_VELOCITIES_ENDPOINT, object_name]),
+                                 timeout=self.__DACE_TIMEOUT, headers=self.__HEADERS)
+        return self._parse_result(response)
```

Comment:
because of the `async_to_sync` method above, you could (and should) rename this function to `query_radial_velocities_async` and have it return `response`.  `async_to_sync` will automatically create a `query_radial_velocities` function that returns `self._parse_result(response)`.

Reply:
Hello,

Thanks for the quick feedback ^^.